src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubConfiguration.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+using Microsoft.Azure.WebJobs.Host.Config;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using System.Threading;
+using Microsoft.ServiceBus.Messaging;
+using Microsoft.Azure.WebJobs.Host.Protocols;
+using System.Globalization;
+using Microsoft.Azure.WebJobs.Host.Triggers;
+
+namespace Microsoft.Azure.WebJobs.ServiceBus
+{
+    public class EventHubConfiguration : IExtensionConfigProvider, IEventHubProvider
```

Comment:
IEventHubProvider is a private interface to the let the binders access the configuration.